tests/test_model_framework.py

```diff
@@ -0,0 +1,49 @@
+import pytz
+import numpy
+
+from pandas import to_datetime
+from tests.test_configurations import SMRFTestCase
+from smrf.framework.model_framework import SMRF
+
+
+class TestModelFramework(SMRFTestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.smrf = SMRF(cls.config_file)
+
+    def test_start_date(self):
+        self.assertEqual(
+            self.smrf.start_date,
+            to_datetime(self.smrf.config['time']['start_date'], utc=True)
```

Comment:
Awesome! This still looks like it's really only testing that UTC works which was the problem before. Maybe duplicate these tests for a non UTC timezone for sanity sake?